configs/body/2d_kpt_sview_rgb_img/deeppose/coco/res50_coco_256x192_rle.py

```diff
@@ -2,11 +2,11 @@
     '../../../../_base_/default_runtime.py',
     '../../../../_base_/datasets/coco.py'
 ]
-evaluation = dict(interval=10, metric='mAP', save_best='AP')
+evaluation = dict(interval=10, metric='mAP', save_best='AP', rle_score=True)
```

Comment:
I think it is not a good idea to set `rle_score` in `evaluation` dict. If possible, let's move it to test_cfg.

Reply:
I see. That's ok for now. The problem is that the rescoring part is currently in `TopDownCocoDataset.evaluate`. Need refactoring in MMPose 2.0, I think.

Reply:
Yes, we can leave this problem to 2.0 refactoring. Currently, we can add it to the dataset.